.zenodo.json

```
@@ -1,6 +1,6 @@
 {
   "title": "mredgebuildings: Prepare data to be used by the EDGE-Buildings model",
-  "version": "0.0.1",
+  "version": "0.0.1.9002",
```

```suggestion
  "version": "0.0.2",
```
The 9000+ development version numbers should not get into the main branch of a pik-piam repository